### PR TITLE
Fixes #2654: Simplified JUnit assertions

### LINKED ISSUE #2654: Simplificable JUnit assertions
**Summary**

There are some JUnit assertions in the codebase which can be simplified using the below simplifications.

Screenshot from 2019-05-25 00-53-27
We can simplify the required JUnit assertions.

### PR DESCRIPTION
**Pull request by RajaVamsi11**
## Description

Simplified the JUnit assertions by replacing the simplifiable assertions.
## Related issues and discussion
Fixes #2654

### CODE REVIEW

**Comment by RajaVamsi11** [comment]
@teolemon @deniger Please review this PR.

File: app/src/test/java/openfoodfacts/github/scrachx/openfood/network/OpenFoodAPIServiceTest.java [hunk 1]
```diff
@@ -286,6 +287,6 @@ private void assertNoProductsFound(Response<Search> response) {
         Search search = response.body();
         List<Product> products = search.getProducts();
         assertTrue(products.isEmpty());
-        assertTrue(Integer.valueOf(search.getCount()) == 0);
+        Assert.assertEquals(0, (int) Integer.valueOf(search.getCount()));
```
Comment (VaiTon) [inline app/src/test/java/openfoodfacts/github/scrachx/openfood/network/OpenFoodAPIServiceTest.java]: Just import `assertEquals` statically

Comment (RajaVamsi11) [reply app/src/test/java/openfoodfacts/github/scrachx/openfood/network/OpenFoodAPIServiceTest.java]: Will do that @VaiTon

Comment (RajaVamsi11) [reply app/src/test/java/openfoodfacts/github/scrachx/openfood/network/OpenFoodAPIServiceTest.java]: Done. Can you please review it again @VaiTon

**Comment by RajaVamsi11** [comment]
@VaiTon Updated according to the changes suggested. Please review it again.

File: app/src/test/java/openfoodfacts/github/scrachx/openfood/network/OpenFoodAPIServiceTest.java [hunk 2]
```diff
@@ -286,6 +287,6 @@ private void assertNoProductsFound(Response<Search> response) {
         Search search = response.body();
         List<Product> products = search.getProducts();
         assertTrue(products.isEmpty());
-        assertTrue(Integer.valueOf(search.getCount()) == 0);
+        assertEquals(0, (int) Integer.valueOf(search.getCount()));
```
Comment (VaiTon) [inline app/src/test/java/openfoodfacts/github/scrachx/openfood/network/OpenFoodAPIServiceTest.java]: Is this cast needed?

Comment (RajaVamsi11) [reply app/src/test/java/openfoodfacts/github/scrachx/openfood/network/OpenFoodAPIServiceTest.java]: Yes @VaiTon

**Pull request merged**